src/hooks/usePageViews.js

```diff
@@ -22,23 +39,20 @@ const ANALYTICS_PROVIDERS = {
   siteimprove: async (location, previousLocation) =>
     /* SiteImprove
     Docs: https://support.siteimprove.com/hc/en-gb/articles/115001615171-Siteimprove-Analytics-Custom-Visit-Tracking
-    Unsure if SiteImprove subscribes to URL changes or not, if they do - we'll fire double events here.
-    Hashrouting support: Unknown, so we follow the same concept as Matomo by pretending there is no hash.
+		Note: siteimprove requires the full URL to track pages, not merely the path
     */
     window._sz &&
     window._sz.push([
       'trackdynamic',
       {
-        url: location.pathname + location.hash.substring(1),
-        ref:
-          (previousLocation && previousLocation.pathname + previousLocation.hash.substring(1)) ||
-          window.location.pathname + window.location.hash.substring(1),
+        url: window.location.origin + location.pathname,
```

Comment:
Can you please re-add the `location.hash.substring(1)` part too?

```suggestion
        url: window.location.origin + location.pathname + location.hash.substring(1),
```